pumpkin/src/client/player_packet.rs

```diff
@@ -442,6 +449,7 @@ impl Player {
                 text_filtering: client_information.text_filtering,
                 server_listing: client_information.server_listing,
             };
+            self.update_client_information().await;
```

Comment:
why here

Reply:
- - This is where we receive the new client information from the player, like skin layers and which hand they are using. So the function is called every time we get new information